docs/src/introduction/faq.md

```diff
@@ -0,0 +1,69 @@
+# FAQ
+
+## Where can I get help?
+
+For usage questions, please contact us via the
+[Julia Discourse](https://discourse.julialang.org/c/domain/opt). If you're
+running into bugs or have feature requests, please use the [GitHub Issue
+Tracker](https://github.com/jump-dev/Convex.jl/issues).
+
+## How does Convex.jl differ from JuMP?
```

Comment:
I think it would make sense to mention the reformulations here. There are some very sophisticated ones ported from https://github.com/hfawzi/cvxquad that can be quite valuable for quantum information theory researchers ([which even cvxpy doesn't have yet](https://ask.cvxr.com/t/can-cvxquad-be-applied-in-python/12277/2)), and having reformulations in general can make some problems simpler to write out

Reply:
I've tried to make very minimal changes to the content of the docs for now. I've added this to https://github.com/jump-dev/Convex.jl/issues/344#issuecomment-1899369803

Reply:
ah somehow I thought this section was new! makes sense